test/support/mocks/dtfs-storage-client.service.mock.ts

```diff
@@ -12,12 +12,16 @@ class MockFileShareClient {
 
 export class MockDtfsStorageClientService {
   readonly baseUrl: string;
+  readonly dealDocumentsFolderPath: string;
   readonly storageSharedKeyCredential: StorageSharedKeyCredential;
   fileShareClient: ShareFileClient;
   getShareFileClient: jest.Mock;
 
   constructor() {
-    (this.baseUrl = null), (this.storageSharedKeyCredential = null), (this.fileShareClient = new MockFileShareClient() as unknown as ShareFileClient);
+    (this.baseUrl = null),
+      (this.dealDocumentsFolderPath = null),
+      (this.storageSharedKeyCredential = null),
+      (this.fileShareClient = new MockFileShareClient() as unknown as ShareFileClient);
```

Comment:
I'm surprised by the brackets and commas here, should this not be
```
this.baseUrl = null;
this.dealDocumentsFolderPath = null;
// etc.
```
?

Reply:
Agreed, I was copying the pattern that was already there (which I don't remember doing myself, so assumed it either came from linting or someone else's refactor which I got when I branched off of main) - have made this change